src/Spatial/Euclidean/Plane.cs

```diff
@@ -74,6 +77,30 @@ public Plane(Point3D rootPoint, UnitVector3D normal)
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Plane"/> struct.
+        /// Constructs a Plane from the given points.
+        /// </summary>
+        /// <param name="points">The points</param>
+        /// <returns>Fitted <see cref="Plane"/></returns>
+        /// <remarks>this method uses SVD to fit the plane.</remarks>
+        public static Plane CreateFittedPlaneFrom(IEnumerable<Point3D> points)
+        {
+            var c = Point3D.Centroid(points);
+
+            var A = CreateMatrix.DenseOfRowVectors(
+                points.Select(p => p - c)
+                    .Select(p => CreateVector.DenseOfArray(new double[] { p.X, p.Y, p.Z })));
+
+            var svd = A.Svd(true);
+            var matV = svd.VT.Transpose();
+            var theIndex = svd.S.Count-1; // in this case, theIndex = 2.
+            var normal = UnitVector3D.OfVector(matV.Column(theIndex));
+
+            var result = new Plane(normal, c);
```

Comment:
`var bestFit = new Plane(normal, c);`

Reply:
I'm afraid that I didn't define any properties the *best* Fit plane should have.
Can I use `fittedPlane` for this?

Reply:
> we should evaluate the best fit based on the orthogonal euclidean distance.

Sorry, I couldn't read all your comments at that moment. my fault.
You had already defined "the best fit" for this problem.
Now, we can use `var bestFit = new Plane(normal, c);`